base_app/src/app/app-tab-overlay/app-tab-overlay.component.ts

```diff
@@ -137,12 +138,13 @@ export class AppTabOverlayComponent implements AfterViewInit, OnInit {
         return nthOverlay;
     }
 
-    getImplementedOverlays(nthOverlay) {
+    getImplementedOverlays(nthOverlay): any {
 
         ClientLogger.log('LogOverlayList', 'getImplementedOverlays(nthOverlay)');
 
         const ajaxRequest = {
-            url: `${this._appURI}php/css_elements_to_json.php?nthOverlay=${nthOverlay}`,
+            // url: `${this._appURI}php/css_elements_to_json.php?nthOverlay=${nthOverlay}`,
+            url: 'http://localhost:4100/bms/php/css_elements_to_json',
```

Comment:
Can't use 4100 for the port - must use the prop-defined value for simpleui-server.  Otherwise, this breaks apps that are using other port numbers. (Unless we've modified the dockerized version of simpleui to always use 4100.

Reply:
fixed with your last commit